render-ws-java-client/src/main/java/org/janelia/render/client/StackAlignmentComparisonClient.java

```diff
@@ -51,35 +55,57 @@ public class StackAlignmentComparisonClient {
 			new StabilizingAffineModel2D<>(new RigidModel2D()), 0.0).createAffineModel2D();
 
 	public static void main(final String[] args) throws Exception {
+		final StackAlignmentComparisonClient client = new StackAlignmentComparisonClient();
+		if (args.length == 0) {
+			final String[] testArgs = {
+					"--baseDataUrl", "http://renderer-dev.int.janelia.org:8080/render-ws/v1",
+					"--owner", "hess_wafer_53",
+					"--project", "cut_000_to_009",
+					"--matchCollection", "c000_s095_v01_match_agg2",
+					"--baselineStack", "c000_s095_v01_align2",
+					"--otherStack", "c000_s095_v01_align_test_xy_ad",
+					"--differenceMetric", "RELATIVE"
+			};
+			client.parse(testArgs);
+		} else {
+			client.parse(args);
+		}
+
 		// data clients (and z values) should be the same for both stacks
-		final RenderDataClient renderClient = new RenderDataClient(baseDataUrl, owner, project);
-		final RenderDataClient matchClient = new RenderDataClient(baseDataUrl, owner, matchCollection);
+		final RenderDataClient renderClient = client.renderParams.getDataClient();
+		final RenderDataClient matchClient = client.matchParams.getMatchDataClient(renderClient.getBaseDataUrl(), renderClient.getOwner());
 
-		final ResolvedTileSpecCollection rtscBaseline = renderClient.getResolvedTilesForZRange(baselineStack, null, null);
-		final ResolvedTileSpecCollection rtscNew = renderClient.getResolvedTilesForZRange(newStack, null, null);
-		final List<CanvasMatches> canvasMatches = getMatchData(matchClient);
+		final List<Double> zValues = renderClient.getStackZValues(client.baselineStack);
+		final AlignmentErrors errorsBaseline = new AlignmentErrors();
+		final AlignmentErrors errorsOther = new AlignmentErrors();
 
-		final AlignmentErrors errorsBaseline = computeSolveItemErrors(rtscBaseline, canvasMatches);
-		final AlignmentErrors errorsNew = computeSolveItemErrors(rtscNew, canvasMatches);
+		for (final Double z : zValues) {
+			final ResolvedTileSpecCollection rtscBaseline = renderClient.getResolvedTiles(client.baselineStack, z);
+			final ResolvedTileSpecCollection rtscOther = renderClient.getResolvedTiles(client.otherStack, z);
+			final List<CanvasMatches> canvasMatchesForZ = getMatchData(matchClient, rtscBaseline);
+
+			errorsBaseline.absorb(computeSolveItemErrors(rtscBaseline, canvasMatchesForZ));
+			errorsOther.absorb(computeSolveItemErrors(rtscOther, canvasMatchesForZ));
+		}
 
-		final AlignmentErrors differences = AlignmentErrors.computeRelativeDifferences(errorsBaseline, errorsNew);
-		AlignmentErrors.writeAsCsv(differences, "pairwiseErrorDifferences.csv");
+		final AlignmentErrors differences = AlignmentErrors.computeDifferences(errorsBaseline, errorsOther, client.differenceMetric.metricFunction);
+		AlignmentErrors.writeToFile(differences, "pairwiseErrorDifferences.json");
```

Comment:
parameterize filename with reasonable default (maybe use .gz suffix since there will be a lot of pairs)